src/quo2/screens/activity_logs.cljs

```diff
@@ -4,71 +4,126 @@
+   "did something here."])
+
+(def complex-user-action
+  [[context-tags/group-avatar-tag "250,000 SNT" {:size :small :color :purple}]
+   "from"
+   [context-tags/group-avatar-tag "Mainnet" {:size :small :color :purple}]
+   "to"
+   [context-tags/group-avatar-tag "Optimism" {:size :small :color :purple}]
+   "on"
+   [context-tags/group-avatar-tag "My savings" {:size :small :color :purple}]])
+
+(def message-with-mention
+  [rn/view {:flex           1
+            :flex-direction :row}
+   [rn/text "Hello"]
+   [rn/text {:style {:color              colors/primary-50
+                     :margin-horizontal  3
+                     :padding-horizontal 3
+                     :background-color   colors/primary-50-opa-10
+                     :border-radius      6}}
+    "@name"]
+   [rn/text "! How are you feeling?"]])
+
 (defn preview []
-  (let [state  (reagent/atom {:title "Activity Title"
-                              :timestamp "Yesterday ∙ 10:41"
-                              :message "Hello Alisher! Do you remember me from the web 3.0 conference in Porto?"
-                              :icon  :placeholder})]
+  (let [state (reagent/atom {:title          "Activity Title"
+                             :timestamp      "Yesterday ∙ 10:41"
+                             :message        :with-mention
+                             :icon           :placeholder
+                             :context        :complex-user-action
+                             :unread?        true
+                             :button-1-label "Reject"
+                             :button-1-type  :danger
+                             :button-2-label "Accept"
+                             :button-2-type  :primary})]
     (fn []
       (let [{:keys [button-1-type
                     button-1-label
                     button-2-type
                     button-2-label]} @state
-            props (cond-> @state
-                    (and (seq button-1-label)
-                         button-1-type)
-                    (assoc :button-1 {:label button-1-label
-                                      :type button-1-type})
+            props                    (cond-> @state
+                                       (and (seq button-1-label)
+                                            button-1-type)
+                                       (assoc :button-1 {:label button-1-label
+                                                         :type  button-1-type})
+
+                                       (and (seq button-2-label)
+                                            button-2-type)
+                                       (assoc :button-2 {:label button-2-label
+                                                         :type  button-2-type})
+
+                                       (= (:message @state) :simple)
+                                       (assoc :message "The quick brown fox jumped over the lazy dog.")
```

Comment:
maybe here it's better to use the same message as in the Figma designs so the designers can verify a bit easier:
i.e "The quick brown fox jumped over the lazy dog."
-> 
"The quick brown fox forgot to jump."